webpack.production.config.js

```diff
@@ -0,0 +1,37 @@
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const path = require("path");
```

Comment:
you do not need one more config just to add uglify js plugin -- you can use environment variables to do this

in webpack config:
`if (process.env.NODE_ENV === 'production') { add plugin  } `

in package.json
`"build cross-env NODE_ENV=production webpack"`


cross-env is a package from npm